Application/EdFi.Ods.Api/Extensions/HttpRequestExtensions.cs

```diff
@@ -7,20 +7,20 @@
 using System.Linq;
 using EdFi.Common.Extensions;
 using EdFi.Ods.Api.Constants;
-using EdFi.Ods.Common.Extensions;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Primitives;
 
 namespace EdFi.Ods.Api.Extensions
 {
     public static class HttpRequestExtensions
     {
-        public static string RootUrl(this HttpRequest request, bool useProxyHeaders = false)
+        public static string RootUrl(this HttpRequest request, bool useProxyHeaders = false, 
```

Comment:
There are a couple of other calling sites that have to specify these new parameters, for example, `OpenApiMetadataController`; since it doesn't pass the parameters, it falls back to the default values and generates incorrect endpoint urls in the metadata (used by Swagger).

I recommend making the parameters mandatory so the compiler shows all the places that need to be updated.

Reply:
I've addressed the other classes that call `RootUrl`.